conf/openmetadata.yaml

```diff
@@ -105,10 +105,10 @@ database:
   # the name of the JDBC driver, mysql in our case
   driverClass: com.mysql.cj.jdbc.Driver
   # the username and password
-  user: openmetadata_user
-  password: openmetadata_password
+  user: ${MYSQL_USER:- openmetadata_user}
+  password: ${MYSQL_USER_PASSWORD:- openmetadata_password}
   # the JDBC URL; the database is called openmetadata_db
-  url: jdbc:mysql://localhost/openmetadata_db?allowPublicKeyRetrieval=true&useSSL=false&serverTimezone=UTC
+  url: jdbc:mysql://${MYSQL_HOST:- localhost}/${MYSQL_DATABASE:- openmetadata_db}?allowPublicKeyRetrieval=true&useSSL=false&serverTimezone=UTC
```

Comment:
I would allow to override the MySQL port too, WDYT?

Reply:
done.